tests/output/vt100_support_posix_err.phpt

```diff
@@ -0,0 +1,140 @@
+--TEST--
+Test stream_vt100_support on POSIX with redirected STDERR
+--SKIPIF--
+<?php
+if (!function_exists('posix_isatty')) {
+    echo "skip Only for POSIX systems";
+}
+?>
+--CAPTURE_STDIO--
+STDERR
+--FILE--
+<?php
+require 'vt100_support.php';
```

Comment:
Fixed in https://github.com/php/php-src/pull/2103/commits/bf60a7880407c92b1b0bace0d1d48b6b5e777b9a